### PR TITLE
Final testing

### LINKED ISSUE #246: [Assessment] Assessment data does not change when clearing/deleting in addressbook
image

### LINKED ISSUE #254: [Assessment] Addassess weird parameter
> Adds an assessment to the addressbook. ParametersL EXAM_NAME ...

Use of EXAM_NAME, and the `...` is not explained

### LINKED ISSUE #257: [Assessment] addgrades weird error message
> Parameters: PERSON_INDEX ASSESSMENT_INDEX Grade

Grade is no upper cased

### LINKED ISSUE #260: [Assessment] addAssessmentStatistics not in help menu

### LINKED ISSUE #262: [Assessment] Grade value inconsistent formatting
> 1. Grades are numeric
> 2. Grade values can only be positive integers (>= 0)

However, when viewing grades, there is a decimal point for some reasons

### LINKED ISSUE #264: [Docu]  deletegrades note can be better worded
> IMPORTANT NOTE: This does not delete grades for the second assessment as per the listing in view grades command, rather for the assessment in second index as seen in list assess command.

### LINKED ISSUE #265: [Assessment] AddStatistics does not add the updated values
1. Add grades to person 1 to assess 1
2. Delete grades to person 1 to assess 1
3. Execute addstatistics 1

Shows outdated values

### LINKED ISSUE #269: [Assessment] null value
1. Add Assess
2. Add Person
3. Add Exam
4. Add grades from Step 1 to Step 2.
5. Register exam from Step 3 to Step 2.
6. ViewAll for Step 2.
image

### PR DESCRIPTION
**Pull request by Meg1456**
Resolves #269  
Resolves #265 
Resolves #264 
Resolves #262 
Resolves #260 
Resolves #257 
Resolves #254 
Resolves #246 

- Edited JavaDoc Comments 
- Fixed Assessments Tests

### CODE REVIEW

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -307,7 +307,7 @@ private static boolean isPrivatePrefixPresent(String matchedPrefix) {
         return new HashSet<>(tagStrings);
     }
     /**
-     * Parses arguments in the context of the EditFeesCommand command.
+     * Parses arguments in the context of the edit fees command
      *
```
Comment (0WN463) [inline src/seedu/addressbook/parser/Parser.java]: Lawl renaming all these makes less sense. We are talking from a technical perspective

Comment (Meg1456) [reply src/seedu/addressbook/parser/Parser.java]: Hahaha I know, but I thought I would make it consistent. Took only a minute to do anyway

Comment (0WN463) [reply src/seedu/addressbook/parser/Parser.java]: I think the Pascal case makes more sense here as we are referring to classes, not random nouns we use everyday

Comment (Meg1456) [reply src/seedu/addressbook/parser/Parser.java]: Oh, I think the original commands in the parser had it like this so I followed the same. I can change it though, no prob. Should I?

Comment (0WN463) [reply src/seedu/addressbook/parser/Parser.java]: I think you should. Feels weird to use those as proper nouns

Comment (Meg1456) [reply src/seedu/addressbook/parser/Parser.java]: Ok!

**Pull request merged**